### DISCUSSION TITLE
Object analytics - clothing color for Human/HumanBody?

### DISCUSSION THREAD

**Discussion by zachvictor** [AxisCommunications/discussions#232] [ACAP]
The [ONVIF Analytics Service Spec](https://www.onvif.org/specs/srv/analytics/ONVIF-Analytics-Service-Spec.pdf) provides for color classification in clothing via HumanBody ([humanbody.xsd](https://www.onvif.org/ver20/analytics/humanbody.xsd)).

Does Axis object analytics provide color classification for clothing? I was hoping to see Color in the metadata stream for "Human" objects, but it seems to appear for vehicle-type objects only. And, I don't find any HumanBody information in objects classified as  Human.

Is it possible to have the device provide color classification for Human objects? Or is there some way to enable HumanBody with color classification enabled for clothing? 

If not, is color classification on the product roadmap (and when might one expect it)? 

Device in question: 
```json
{
    "apiVersion": "1.3",
    "data": {
        "propertyList": {
            "Architecture": "aarch64",
            "ProdNbr": "P3267-LVE",
            "ProdFullName": "AXIS P3267-LVE Dome Camera",
            "Version": "11.3.70",
            "ProdType": "Dome Camera",
            "Soc": "Axis Artpec-8",
            "Brand": "AXIS",
            "ProdShortName": "AXIS P3267-LVE",
            "BuildDate": "Mar 16 2023 19:08"
        }
    }
}
```

**Comment by sguaqueta** [comment]
Hi @zachvictor, 
Thanks for your question. This functionality is currently being developed and will be available in an upcoming firmware release. I'd be more than happy to share additional details and I'm also highly interested in learning more about the application/use-case that you are currently working on so I'll contact you separately to discuss further.

**Comment by vivekatoffice** [comment]
Related link:
[Adding upper and lower clothing color in the metadata stream](https://www.axis.com/developer-community/news/clothing-color)
